pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, cairo
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  version = "eacbbc837e";
```

Comment:
Please use a proper version or `unstable-$commitDate` (I think this is current consensus)